src/utils/priceFormat.ts

```diff
@@ -0,0 +1,16 @@
+
+const priceFormat = (tilePrice: number, deliveryPrice: number, installPrice: number ) => {
+  let totalPrice = (tilePrice + deliveryPrice + installPrice).toFixed(2);

```

Comment:
if this function is named price format, then this function should only do one thing which is formatting the price, the sum of total price should not be included.